InChIKey property (#466)
Materials-Consortia/OPTIMADE: optimade.rst

  - A filter that matches an exactly given formula is :filter:`chemical_formula_anonymous="A2B"`.

inchikey
~~~~~~~~

> - **Description**: The standard InChIKey representation of the structure, as laid out by the [InChI Trust](https://www.inchi-trust.org)

Reviewer: ```suggestion
- **Description**: The standard InChIKey representation of the structure, as laid out by the `InChI Trust <https://www.inchi-trust.org>`_
```

Sorry, used the wrong markup...

> - **Description**: The standard InChIKey representation of the structure, as laid out by the `InChI Trust <https://www.inchi-trust.org>`_

Reviewer 2: InChIKey is more of a non-unique identifier than a representation of a structure since it is constructed using a one way function (there is no way to programmatically restore the structure from an InChIKey). Wikipedia also states that InChIKey is a digital representation of the InChi, not the structure (whatever "digital representation" means).

Author: I am for replacing "representation" with an "identifier". I recall there were situations when non-unique InChIKeys were produced, but I guess that depends on the producing software which performs intermediate transformations before feeding the structure to the InChI library. Should we put a caution in the description?

Author: Right, adding a disclaimer for collisions makes sense.

> - **Description**: The standard InChIKey representation of the structure, as laid out by the `InChI Trust <https://www.inchi-trust.org>`_

Reviewer 2: Maybe we should also specify which InChI/InChiKey version should be used?

Author: Fair point. I would also like to discuss how will we handle deprecation and switching to new InChiKey versions in the future, as I do not believe it will be possible to keep using the old version when new ones are released. I can think of the following options:

1. When InChiKey version changes, put the newest version in the upcoming OPTIMADE specification vX and require all implementers of vX to recompute their InChiKey.
2. Define the InChiKey version as metadata (after #462 gets solved).

First solution is better for interoperability, but puts more strain on the implementers.

  - A filter that matches an exactly given formula is :filter:`chemical_formula_anonymous="A2B"`.

inchikey
~~~~~~~~

> - **Description**: The standard InChIKey identifier of the structure, as laid out by the `InChI Trust <https://www.inchi-trust.org>`_

Reviewer 2: ```suggestion
- **Description**: The standard InChIKey identifier of the structure, as laid out by the `InChI Trust <https://www.inchi-trust.org>`_.
```

>   Standard InChIKey is non-unique, thus the same InChIKey can be assigned to several different structures.

Reviewer 2: ```suggestion
  Standard InChIKey is not guaranteed to be unique and on extremely rare the same InChIKey may be assigned to several different structures.
```

>   Standard InChIKey is non-unique, thus the same InChIKey can be assigned to several different structures.

Reviewer 3: From the workshop: must be clarified if the key MUST cover all parts of the structure or not. Consider if this should be a list so that components can be covered, or if that goes in a separate field. (May need careful definition of what a component/fragment is).
